packages/app/src/app/pages/Patron/PricingModal/PricingChoice/ChangeSubscription/index.tsx

```diff
@@ -17,20 +17,24 @@ interface Props {
   markedAsCancelled: boolean;
   cancelSubscription: () => void;
   updateSubscription: (params: { coupon: string }) => void;
-  store: any;
-  signals: any;
 }
 
 function ChangeSubscriptionComponent({
   date,
   markedAsCancelled,
   cancelSubscription,
   updateSubscription,
-  store,
-  signals,
 }: Props) {
-  const isLoading = store.patron.isUpdatingSubscription;
-  const { error } = store.patron;
+  const {
+    state: {
+      patron: { isUpdatingSubscription, error },
+    },
+    actions: {
+      modalOpened,
+      patron: { tryAgainClicked },
+    },
+  } = useOvermind();
+  const isLoading = isUpdatingSubscription;
```

Comment:
How about not defining a new variable and directly using `isUpdatingSubscription` or doing this `{ isUpdatingSubscription: isLoading, error }` at `L30`?

Reply:
Wdyt @Saeris @christianalfoni  ?

Reply:
> How about not defining a new variable and directly using `isUpdatingSubscription` or doing this `{ isUpdatingSubscription: isLoading, error }` at `L30`?

okay, will fix it.

Reply:
@jyash97 its fixed, and also rebase commit. let me know if any more changes are required.